src/main/resources/org/jenkinsci/plugins/saml/SamlSecurityRealm/config.jelly

```diff
@@ -16,5 +16,36 @@
     <f:entry title="Maximum Authentication Lifetime" field="maximumAuthenticationLifetime" help="/plugin/saml/help/maximumAuthenticationLifetime.html">
       <f:textbox default="86400" />
     </f:entry>
+    <f:entry title="Username Attribute"  field="usernameAttributeName" 
+             help="/plugin/saml/help/usernameAttributeName.html">
+      <f:textbox />
+    </f:entry>
+    
+    <f:optionalBlock title="Use Encryption ?" 
```

Comment:
The **f:entry=>table=>f:optionalBlock** chain can help here to force this block to hide optional part on initial load, see [this thread](http://jenkins-ci.361315.n4.nabble.com/How-do-I-show-other-configuration-entries-only-if-the-user-have-checked-a-checkbox-in-config-jelly-td388471.html)